tinyagentos/routes/github_oauth.py

```
@@ -201,3 +201,237 @@ async def delete_identity(request: Request, identity_id: str):
+            "https://api.github.com/user",
+            headers={
+                "Authorization": f"Bearer {token}",
+                "Accept": "application/vnd.github+json",
+            },
+            timeout=10,
+        )
+        user_resp.raise_for_status()
+        # For installations, the "user" endpoint returns the GitHub App bot user.
+        # We need the actual installing account. Use /app/installations/{id} instead.
+    except Exception:
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)